src/main/java/pixelitor/gui/utils/SliderSpinner.java

```diff
@@ -59,17 +63,28 @@
     private boolean spinnerMoved = false;
 
     public SliderSpinner(RangeParam model, TextPosition position, boolean addDefaultButton) {
-        this(model, null, null, position, addDefaultButton);
+        this(model, position, addDefaultButton, HORIZONTAL);
```

Comment:
Yeah, I did a mess with the constructors 😅.
:wasn't sure how should I resolve that...